corehq/ex-submodules/pillowtop/migrations/0003_auto_20170411_1957.py

```diff
@@ -11,7 +11,9 @@ class Migration(migrations.Migration):
     dependencies = [
         ('pillowtop', '0002_djangopillowcheckpoint_sequence_format'),
     ]
-
+    run_before = [
+        ('pillowtop', '0004_offset_to_big_int'),
+    ]
```

Comment:
Why are these `run_before` necessary?

Reply:
I was trying to fix this https://github.com/dimagi/commcare-hq/pull/21629#discussion_r222178272. Still not able to figure out why travis is not picking up the  `0005_kafkacheckpoint_doc_modification_time`.